x-pack/test/functional/apps/maps/group1/vector_styling.js

```diff
@@ -8,7 +8,8 @@
 import expect from '@kbn/expect';
 
 export default function ({ getService, getPageObjects }) {
-  const PageObjects = getPageObjects(['common', 'maps', 'timePicker']);
+  const PageObjects = getPageObjects(['maps', 'timePicker', 'common']);
```

Comment:
How do these changes move toward the goal of using ui settings instead of timepicker ui. This change just looks like re-ordered page objects array. Are there missing changes?

Reply:
@nreese tagged you in code changes. GH messed up? or I did. Not sure. Thanks!